tests/php/Security/MemberAuthenticator/SessionAuthenticationHandlerTest.php

```diff
@@ -15,10 +15,6 @@ class SessionAuthenticationHandlerTest extends SapphireTest
 
     protected $usesDatabase = true;
 
-    /**
-     * @runInSeparateProcess
```

Comment:
These annotations were removed because they will cause phpunit to fail on any deprecations warnings e.g. swiftmailer is deprecated.  They did not seem to be required to be there for the unit tests to pass, so I simply removed them